libra_toolbox/neutron_detection/activation_foils/compass.py

```diff
@@ -23,4 +25,33 @@ def get_channel(filename):
     return int(filename.split('@')[0][7:])
 
 
+def sort_compass_files(directory):
```

Comment:
Let's try and do some type hinting here cause i initially got confused:

```suggestion
def sort_compass_files(directory: str) -> dict:
```